execute-test-cases.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Execute your tests
+description: "Run your tests to find bugs in your app. We will help you figure out what went wrong."
+icon: "play"
+---
+
+## Run your active test cases
+
+You should have number of **active tests** in your test suite by now. They are `on`.
+
+To check if there are any bugs in your new build or in production preventing your users from operating your app, you will have to **run the tests**.
+
+### 1. Trigger a test run
+
+To run all active tests instantly click `run all` and then `in app`. This will trigger a test run resulting in a [test report](/test-reports.mdx) visible in the Octomind app. We run the tests in parallel to make sure they run completes under 20 minutes, regadless of the number of your active test cases.
+
+Useful for **branch deployments** - you can run your tests against any other accessible URL. Select `in app on a different URL` instead.
+
+<Frame caption="Run your tests for the first time - screenshot 07/2024">
+  <img
+    src="/images/setup/setup-9-run-tests.png"
+    alt="Create your first test report - screenshot 07/2024"
+  />
+</Frame>
+
+<div class="mt-8" />
+
+<Warning>
+  We also **"run"** a test cases when you save changes to a test you have
```

Comment:
We will give you the option to do that with the new save & run button that is in the works, currently you'd need to execute it yourself manually (unless you do re-generate steps which runs automatically)

Reply:
I know. I'm being a bit ahead.